Riot/Modules/Room/RoomViewController.m

```diff
@@ -4804,28 +5032,34 @@ - (void)roomInputToolbarViewDidOpenActionMenu:(RoomInputToolbarView*)toolbarView
 
 - (void)roomInputToolbarView:(RoomInputToolbarView *)toolbarView sendAttributedTextMessage:(NSAttributedString *)attributedTextMessage
 {
-    BOOL isMessageAHandledCommand = NO;
-    // "/me" command is supported with Pills in RoomDataSource.
-    if (![attributedTextMessage.string hasPrefix:kMXKSlashCmdEmote])
-    {
-        // Other commands currently work with identifiers (e.g. ban, invite, op, etc).
-        NSString *message;
-        if (@available(iOS 15.0, *))
+    // Re-invite the left member before sending the message in case of a discussion (direct chat)
+    MXWeakify(self);
+    [self createOrRestoreDiscussionIfNeeded:^(BOOL success) {
+        MXStrongifyAndReturnIfNil(self);
+        
+        BOOL isMessageAHandledCommand = NO;
+        // "/me" command is supported with Pills in RoomDataSource.
+        if (![attributedTextMessage.string hasPrefix:kMXKSlashCmdEmote])
         {
-            message = [PillsFormatter stringByReplacingPillsIn:attributedTextMessage mode:PillsReplacementTextModeIdentifier];
+            // Other commands currently work with identifiers (e.g. ban, invite, op, etc).
+            NSString *message;
+            if (@available(iOS 15.0, *))
+            {
+                message = [PillsFormatter stringByReplacingPillsIn:attributedTextMessage mode:PillsReplacementTextModeIdentifier];
+            }
+            else
+            {
+                message = attributedTextMessage.string;
+            }
+            // Try to send the slash command
```

Comment:
we have to check the DM creation when the first message is a command...

Reply:
The command is sent after DM was created. I tested `/me` command without problem.